Travel-Planner/Travel-Planner/Screens/TripListView.swift

```diff
@@ -10,22 +10,73 @@ import SwiftUI
 struct TripListView: View {
     @StateObject var vm = TripViewModel()
     @State private var showAddTrip: Bool = false
-    
+    @State private var isEditing: Bool = false
+    @State private var showDeleteConfirmation = false
+    @State private var tripToDelete: TripEntity?
+
     var body: some View {
         NavigationStack {
-            ScrollView{
+            ScrollView {
                 LazyVStack {
                     ForEach(vm.trips) { trip in
-                        NavigationLink(destination: TripView(vm: DayPlanViewModel(trip: trip))) {
-                            Text(trip.title)
+                        if isEditing {
+                            // Deletable trip row
+                            HStack {
```

Comment:
is there a way to make this into a list? SwiftUI has system function to edit and delete item in a list